CHANGES

```diff
@@ -75,6 +75,9 @@ Bugs fixed
 
 * #11079: LaTeX: figures with align attribute may disappear and strangely impact
   following lists
+* #11093: LaTeX: fix "multiply-defined references" PDF build warnings when one or
+  more reST labels directly precede an :rst:dir:`py:module` or :rst:dir:`automodule`
```

Comment:
I am not native speaker so I don't know if we should use singular `precedes` or plural as here

Reply:
ok at least we have some argument now! I will merge after testing completes thanks in particular for very thorough added test on this occasion